azure-pipelines.yml

```diff
@@ -7,8 +7,20 @@ jobs:
         versionSpec: '3.8'
     - bash: |
         python -m pip install -r requirements-dev.txt
+        python -m pip install pytest-azurepipelines
         python ./bin/run_tests.py
 
+- job: linux_pytest
+  pool: {vmImage: 'Ubuntu-18.04'}
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.8'
+    - bash: |
+        python -m pip install -r requirements-dev.txt
+        python -m pip install pytest-azurepipelines
+        pytest unit_test test
+
```

Comment:
Why did you add this whole section ? Just to show `pytest` direct command invocation ?

Reply:
Current pipeline work is to call test by `python ./bin/run_tests.py`. When my changes are applied the whole content of bin directory can be removed, but from the other side its need change of workflow or all contributor.  So I implement this in conservative way, so show how to do this, but allow @joerick decide which approach is best for him.

Reply:
If there's just the old and the new version in the diff, @joerick would also have noticed when reviewing the changes.